com.unity.uiwidgets/Runtime/widgets/framework.cs

```diff
@@ -2089,14 +2354,54 @@ public abstract class Element : DiagnosticableTree, BuildContext {
 
     public delegate Widget ErrorWidgetBuilder(UIWidgetsErrorDetails details);
 
+    internal class _ElementDiagnosticableTreeNode : _DiagnosticableTreeNode {
+        internal _ElementDiagnosticableTreeNode(
+            Element value,
+            DiagnosticsTreeStyle style,
+            string name = null,
+            bool stateful = false
+        ) : base(
+            name: name,
+            value: value,
+            style: style
+        ) {
+            this.stateful = stateful;
+        }
+
+        readonly bool stateful;
+
+        public override Dictionary<string, object> toJsonMap() {
+            Dictionary<string, object> json = base.toJsonMap();
+            Element element = value as Element;
+            json["widgetRuntimeType"] = element.widget?.GetType()?.ToString();
+            json["stateful"] = stateful;
+            return json;
+        }
+    }
+
     public class ErrorWidget : LeafRenderObjectWidget {
-        public ErrorWidget(Exception exception) : base(key: new UniqueKey()) {
-            message = _stringify(exception);
+        public ErrorWidget(Exception exception, string message = null) : base(key: new UniqueKey()) {
+            this.message = message ?? _stringify(exception);
+            _uiWidgetsError = exception is UIWidgetsError uiWidgetsError ? uiWidgetsError : null;
+        }
+
+        public static ErrorWidget withDetails(string message = "", UIWidgetsError error = null
+        ) {
+            return new ErrorWidget(error, message);
         }
 
         public static ErrorWidgetBuilder builder = (details) => new ErrorWidget(details.exception);
 
-        public readonly string message;
+        static Widget _defaultErrorWidgetBuilder(UIWidgetsErrorDetails details) {
+            string message = "";
+            D.assert(() => {
+                message = _stringify(details.exception) + "\nSee also: https://flutter.dev/docs/testing/errors";
```

Comment:
we should not navigate users to flutter directly in our code.  please remove the "\nSee also: ,..." line.